tests/globalhelper/reporthelper.go

```diff
@@ -232,3 +232,19 @@ func containsString(list []string, item string) bool {
 
 	return false
 }
+
+func CopyClaimFileToTcFolder(tcName, formattedTcName string) {
+	srcClaim := path.Join(Configuration.General.TnfReportDir, globalparameters.DefaultClaimFileName)
+	dstClaim := path.Join(Configuration.General.ReportDirAbsPath, "Debug", getTestSuiteName(tcName), formattedTcName,
+		globalparameters.DefaultClaimFileName)
+
+	if _, err := os.Stat(srcClaim); err == nil {
```

Comment:
Minor suggestion to avoid nested "if"s:
```
_, err := err := os.Stat(srcClaim)
if err != nil {
    a) log(...)
    b) return fmt.Errorf() -> only if function is updated to return errors.
}

err = CopyFiles(srcClaim, dstClaim)
if err != nil {
   a) log(...)
   b) return fmt.Errorf(...) -> only if function is updated to return errors.
}

return (nil)
```

Reply:
thanks